test/e2e/framework/main_entry.go

```diff
@@ -0,0 +1,23 @@
+package framework
```

Comment:
Could we call this file `main_test.go` and rename `MainEntry` to `MainTest`?

Reply:
I am planning to create a MainTest file outside of framework package that calls MainEntry like this: https://github.com/coreos/etcd-operator/blob/master/test/e2e/main_test.go

Reply:
Ah got it, makes sense.